src/chocolatey/infrastructure.app/services/ChocolateyConfigSettingsService.cs

```diff
@@ -75,29 +79,35 @@ public virtual IEnumerable<ChocolateySource> ListSources(ChocolateyConfiguration
                     if (configuration.RegularOutput)
                     {
                         this.Log().Info(() => "{0}{1} - {2} {3}| Priority {4}|Bypass Proxy - {5}|Self-Service - {6}|Admin Only - {7}.".FormatWith(
-                        source.Id,
-                        source.Disabled ? " [Disabled]" : string.Empty,
-                        source.Value,
-                        (string.IsNullOrWhiteSpace(source.UserName) && string.IsNullOrWhiteSpace(source.Certificate)) ? string.Empty : "(Authenticated)",
-                        source.Priority,
-                        source.BypassProxy.ToStringSafe(),
-                        source.AllowSelfService.ToStringSafe(),
-                        source.VisibleToAdminsOnly.ToStringSafe()
-                        ));
+                            source.Id,
+                            source.Disabled ? " [Disabled]" : string.Empty,
+                            source.Value,
+                            (string.IsNullOrWhiteSpace(source.UserName) && string.IsNullOrWhiteSpace(source.Certificate)) ? string.Empty : "(Authenticated)",
+                            source.Priority,
+                            source.BypassProxy.ToStringSafe(),
+                            source.AllowSelfService.ToStringSafe(),
+                            source.VisibleToAdminsOnly.ToStringSafe()
+                            ));
                     }
                     else
                     {
-                        this.Log().Info(() => "{0}|{1}|{2}|{3}|{4}|{5}|{6}|{7}|{8}".FormatWith(
-                        source.Id.QuoteIfContainsPipe(),
-                        source.Value,
-                        source.Disabled.ToStringSafe(),
-                        source.UserName.QuoteIfContainsPipe(),
-                        source.Certificate,
-                        source.Priority,
-                        source.BypassProxy.ToStringSafe(),
-                        source.AllowSelfService.ToStringSafe(),
-                        source.VisibleToAdminsOnly.ToStringSafe()
-                        ));
+                        if (configuration.IncludeHeaders && !hasHeaderRowBeenOutput)

```

Comment:
Any reason why this is a standalone if statement, instead of being combine with the else statement as `else if`?

Reply:
This needs to be here, since the `else`above is for outputing the actual source, whereas the new `if` statement is for the header row, if the header row is required.